packages/terra-form-select/src/SingleSelect.jsx

```diff
@@ -170,41 +166,40 @@ class SingleSelect extends React.Component {
 
   render() {
     const {
-      allowClear,
       children,
       defaultValue,
       onChange,
-      placeholder,
       required,
       value,
       intl,
+      isFilterStyle,
       ...otherProps
     } = this.props;
 
-    const defaultPlaceholder = intl.formatMessage({ id: 'Terra.form.select.defaultDisplay' });
-    const selectPlaceholder = placeholder === undefined ? defaultPlaceholder : placeholder;
-    let clearOptionDisplay;
+    let selectValue;
 
-    if (allowClear) {
-      if (selectPlaceholder.length === 0) {
-        clearOptionDisplay = defaultPlaceholder;
+    if (!isFilterStyle || this.state.value || value) {
```

Comment:
This will need to account for falsey, but real, values. empty string and the number zero for example. Generally these are done by checking for undefined.

Reply:
updated condition here https://github.com/cerner/terra-core/pull/3094/commits/5c0d19afda050f822e2d6e7fcd04abefa15886b6